src/script/stack.zig

```diff
@@ -87,7 +87,7 @@ pub const Stack = struct {
 
         if (value.len > 8) return StackError.InvalidValue;
 
-        var buffer: [8]u8 = undefined;
+        var buffer: [8]u8 = [_]u8{0, 0, 0, 0, 0, 0, 0, 0};
         std.mem.copyBackwards(u8, buffer[0..value.len], value);
```

Comment:
you can use memcopy there

Reply:
@tdelabro can you check now from the updates. Is that what you mean?